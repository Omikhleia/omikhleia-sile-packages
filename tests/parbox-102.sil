\begin[papersize=a6, class=book]{document}% Needs book class for footnotes
\neverindent
\nofolios
\set[parameter=document.parskip, value=5pt]
\script[src=packages/parbox]
\script[src=packages/lorem]
% An end-to-end test for parbox inception and footnotes cascading,
% covering also parbox minimization.

% No minimization
A. \parbox[width=70%fw, strut=character, padding=5pt, border=0.5pt]{%
I am a parbox\footnote{Main parbox footnote A.}

\parbox[width=70%fw, strut=character, padding=5pt, border=0.5pt]{%
I am an insider parbox\footnote{Insider parbox footnote A.}}
}

% Minimize outer frame
B. \parbox[width=70%fw, strut=character, padding=5pt, border=0.5pt, minimize=true]{%
I am a parbox\footnote{Main parbox footnote B.}

\parbox[width=70%fw, strut=character, padding=5pt, border=0.5pt]{%
I am an insider parbox\footnote{Insider parbox footnote B.}}
}

% Minimize inner frame too.
C. \parbox[width=70%fw, strut=character, padding=5pt, border=0.5pt, minimize=true]{%
I am a parbox\footnote{Main parbox footnote C.}

\parbox[width=70%fw, strut=character, padding=5pt, border=0.5pt, minimize=true]{%
I am an insider parbox\footnote{Insider parbox footnote C.}}
}

\end{document}