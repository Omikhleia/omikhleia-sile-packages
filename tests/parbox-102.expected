Set paper size 	297.6377985	419.5275636
Begin page
Mx 	24.7039
My 	61.8195
Set font 	Gentium Plus;10;400;;normal;;LTR
T	36 w=5.9521	(A)
Mx 	30.6561
T	17 w=2.2900	(.)
Draw line	35.5940	48.6652	171.8852	0.5000
Draw line	35.5940	95.7589	171.8852	0.5000
Draw line	35.5940	48.6652	0.5000	47.5938
Draw line	207.4792	48.6652	0.5000	47.5938
Mx 	40.5940
T	44 w=2.9785	(I)
Mx 	46.2263
T	68 80 w=12.6221	(am)
Mx 	61.5021
T	68 w=4.5898	(a)
Mx 	68.7458
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	97.7301
My 	58.6408
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	20 w=3.1962	(1)
Draw line	40.5940	70.1632	123.3196	0.5000
Draw line	40.5940	90.7589	123.3196	0.5000
Draw line	40.5940	70.1632	0.5000	21.0957
Draw line	163.9137	70.1632	0.5000	21.0957
Mx 	45.5940
My 	83.3175
Set font 	Gentium Plus;10;400;;normal;;LTR
T	44 w=2.9785	(I)
Mx 	51.2169
T	68 80 w=12.6221	(am)
Mx 	66.4834
T	68 81 w=10.1074	(an)
Mx 	79.2352
T	76 81 86 76 71 72 85 w=28.5791	(insider)
Mx 	110.4586
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	139.4430
My 	80.1388
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	21 w=3.1962	(2)
Mx 	24.7039
My 	114.9132
Set font 	Gentium Plus;10;400;;normal;;LTR
T	37 w=5.4980	(B)
Mx 	30.2020
T	17 w=2.2900	(.)
Draw line	35.1442	101.7589	133.3196	0.5000
Draw line	35.1442	148.8527	133.3196	0.5000
Draw line	35.1442	101.7589	0.5000	47.5938
Draw line	168.4639	101.7589	0.5000	47.5938
Mx 	40.1442
T	44 w=2.9785	(I)
Mx 	45.7723
T	68 80 w=12.6221	(am)
Mx 	61.0439
T	68 w=4.5898	(a)
Mx 	68.2832
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	97.2676
My 	111.7345
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	22 w=3.1962	(3)
Draw line	40.1442	123.2570	123.3196	0.5000
Draw line	40.1442	143.8527	123.3196	0.5000
Draw line	40.1442	123.2570	0.5000	21.0957
Draw line	163.4639	123.2570	0.5000	21.0957
Mx 	45.1442
My 	136.4113
Set font 	Gentium Plus;10;400;;normal;;LTR
T	44 w=2.9785	(I)
Mx 	50.7671
T	68 80 w=12.6221	(am)
Mx 	66.0336
T	68 81 w=10.1074	(an)
Mx 	78.7854
T	76 81 86 76 71 72 85 w=28.5791	(insider)
Mx 	110.0088
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	138.9932
My 	133.2326
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	23 w=3.1962	(4)
Mx 	24.7039
My 	168.0070
Set font 	Gentium Plus;10;400;;normal;;LTR
T	38 w=5.3613	(C)
Mx 	30.0653
T	17 w=2.2900	(.)
Draw line	35.0093	154.8527	117.0380	0.5000
Draw line	35.0093	201.9464	117.0380	0.5000
Draw line	35.0093	154.8527	0.5000	47.5938
Draw line	152.0473	154.8527	0.5000	47.5938
Mx 	40.0093
T	44 w=2.9785	(I)
Mx 	45.6356
T	68 80 w=12.6221	(am)
Mx 	60.9054
T	68 w=4.5898	(a)
Mx 	68.1429
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	97.1273
My 	164.8283
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	24 w=3.1962	(5)
Draw line	40.0093	176.3507	107.0380	0.5000
Draw line	40.0093	196.9464	107.0380	0.5000
Draw line	40.0093	176.3507	0.5000	21.0957
Draw line	147.0473	176.3507	0.5000	21.0957
Mx 	45.0093
My 	189.5050
Set font 	Gentium Plus;10;400;;normal;;LTR
T	44 w=2.9785	(I)
Mx 	50.6304
T	68 80 w=12.6221	(am)
Mx 	65.8951
T	68 81 w=10.1074	(an)
Mx 	78.6451
T	76 81 86 76 71 72 85 w=28.5791	(insider)
Mx 	109.8667
T	83 68 85 69 82 91 w=28.9844	(parbox)
Mx 	138.8511
My 	186.3263
Set font 	Gentium Plus;6.81152;400;;normal;;LTR
T	25 w=3.1962	(6)
Mx 	24.7039
My 	290.5944
Set font 	Gentium Plus;9;400;;normal;;LTR
T	20 w=4.2231	(1)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	48 68 76 81 w=18.7910	(Main)
Mx 	70.1689
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	98.6447
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	132.7322
T	36 w=5.3569	(A)
Mx 	138.0892
T	17 w=2.0610	(.)
Mx 	24.7039
My 	301.9469
T	21 w=4.2231	(2)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	44 81 86 76 71 72 85 w=25.9629	(Insider)
Mx 	77.3401
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	105.8151
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	139.9020
T	36 w=5.3569	(A)
Mx 	145.2589
T	17 w=2.0610	(.)
Mx 	24.7039
My 	313.2995
T	22 w=4.2231	(3)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	48 68 76 81 w=18.7910	(Main)
Mx 	70.1690
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	98.6447
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	132.7323
T	37 w=4.9482	(B)
Mx 	137.6806
T	17 w=2.0610	(.)
Mx 	24.7039
My 	324.6520
T	23 w=4.2231	(4)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	44 81 86 76 71 72 85 w=25.9629	(Insider)
Mx 	77.3401
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	105.8152
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	139.9021
T	37 w=4.9482	(B)
Mx 	144.8503
T	17 w=2.0610	(.)
Mx 	24.7039
My 	336.0045
T	24 w=4.2231	(5)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	48 68 76 81 w=18.7910	(Main)
Mx 	70.1690
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	98.6448
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	132.7324
T	38 w=4.8252	(C)
Mx 	137.5576
T	17 w=2.0610	(.)
Mx 	24.7039
My 	347.3571
T	25 w=4.2231	(6)
Mx 	28.9271
T	17 w=2.0610	(.)
Mx 	48.9881
T	44 81 86 76 71 72 85 w=25.9629	(Insider)
Mx 	77.3401
T	83 68 85 69 82 91 w=26.0859	(parbox)
Mx 	105.8152
T	73 82 82 87 81 82 87 72 w=31.6978	(footnote)
Mx 	139.9021
T	38 w=4.8252	(C)
Mx 	144.7273
T	17 w=2.0610	(.)
End page
Finish
